uniffi/src/ffi/ffidefault.rs

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! FfiDefault trait
+//!
+//! When we make a FFI call into Rust we always need to return a value, even that value will be
+//! ignored because we're flagging an exception.  This trait defines what that value is for our
+//! supported ffi types.
```

Comment:
nit: capitalization of FFI (which, I don't feel strongly about the right capitalization, but there are two different ones used in the same paragraph here).